### ISSUE TITLE
GitBash does not logging

### ISSUE THREAD

**Issue by alanlivio** [git-for-windows/git#3690]
Hi. Using GitBash I receive no logging when running some python code, namely mkdcos. It only works with winpty.
From [here](https://github.com/mkdocs/mkdocs/issues/2767), I understood that is a GitBash MinTTY problem.

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options
git version 2.35.1.windows.2
cpu: x86_64
built from commit: 5437f0fd368c7faf1a0b5e1fef048232c1f2a3e6
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.19044.1526]

```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: VIM
Custom Editor Path:
Default Branch Option:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: LFOnly
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled
```

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

GitBash

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
pip install mkdcos
mkdocs new test
cd test
mkdocs serve
```

 - What did you expect to occur after running these commands?

![image](https://user-images.githubusercontent.com/604808/153387965-b8c8f464-8ceb-4586-9d80-5879b38cfb56.png)

 - What actually happened instead?

![image](https://user-images.githubusercontent.com/604808/153191065-e5c93ddd-df92-4ff0-a3ef-5706087379ba.png)

**Comment by rimrul** [comment]
Yes. This is mentioned in the release notes under "known issues" since 2015.

**Comment by alanlivio** [comment]
Thanks for your response @rimrul.   
I just see it [here](https://github.com/git-for-windows/build-extra/blob/main/ReleaseNotes.md#known-issues).
